modules/core/src/utils/json-loader.js

```diff
@@ -0,0 +1,12 @@
+function isJSON(text) {
```

Comment:
Some notes:

- There is already JSON loader in loaders.gl/experimental. It was hard to justify its own module for < 10 lines... thought about placing it in core. 
- An option would be to simply adding a `json` flag to the loader objects (in addition to `text` and `binary`) and build in json parsing to `load` and `parse`.
- The there would also be the option of offering a version of the JSON parser that gives meaningful errors. We could offer that as a "pluggable" module.

Reply:
I prefer it in core. That will save us a new dependency.